### ISSUE TITLE
develop: Broken layout of embedded TabContainer when using Append

### ISSUE THREAD

**Issue by jeandeaual** [fyne-io/fyne#828]
Hello,

I noticed a regression with tabs using the develop branch (v1.2.4-0.20200406152143-ddca854edfa1 in go.mod).

Using the following example:

```go
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	app := app.New()
	win := app.NewWindow("Window")
	win.SetMaster()

	tabs := widget.NewTabContainer()
	tabs.Append(
		widget.NewTabItem("Tab 1", widget.NewVBox(
			widget.NewRadio([]string{
				"A",
				"B",
				"C",
			}, nil),
			widget.NewTabContainer(
				widget.NewTabItem(
					"Test Inner 1-1",
					widget.NewVBox(widget.NewLabel("Test Inner 1-1")),
				),
				widget.NewTabItem(
					"Test Inner 1-2",
					widget.NewVBox(widget.NewLabel("Test Inner 1-2")),
				),
			),
		)),
	)
	tabs.Append(
		widget.NewTabItem("Tab 2", widget.NewVBox(
			widget.NewRadio([]string{
				"D",
				"E",
			}, nil),
			widget.NewTabContainer(
				widget.NewTabItem(
					"Test Inner 2-1",
					widget.NewVBox(widget.NewLabel("Test Inner 2-1")),
				),
				widget.NewTabItem(
					"Test Inner 2-2",
					widget.NewVBox(widget.NewLabel("Test Inner 2-2")),
				),
			),
		)),
	)

	win.SetContent(tabs)
	win.ShowAndRun()
}
```

The resulting windows looks like this:

![fyne-develop-tab-issue](https://user-images.githubusercontent.com/23304955/78785062-6b1ebb00-79e1-11ea-953e-5bb9f5a1b210.png)

After clicking on one of the tabs, this gets fixed:

![fyne-develop-tab-after-click](https://user-images.githubusercontent.com/23304955/78785151-95707880-79e1-11ea-8862-5aa269bd6ecc.png)

Note that the radio buttons don't appear correctly. This only seems to happen on Windows (at least the radio buttons were correct when I tried on ArchLinux).

The same code with Fyne 1.2.3:

![fyne-1 2 3](https://user-images.githubusercontent.com/23304955/78785104-7c67c780-79e1-11ea-8c91-4e8dc7ae1f43.png)

However, when we add the tab items directly in `NewTabContainer` instead of using `Append`, the same issue doesn't happen (with both 1.2.3 and ed30d6786124c2c34e9d454076901df3e3830ca8):

```go
package main

import (
	"fyne.io/fyne/app"
	"fyne.io/fyne/widget"
)

func main() {
	app := app.New()
	win := app.NewWindow("Window")
	win.SetMaster()

	tabs := widget.NewTabContainer(
		widget.NewTabItem("Tab 1", widget.NewVBox(
			widget.NewRadio([]string{
				"A",
				"B",
				"C",
			}, nil),
			widget.NewTabContainer(
				widget.NewTabItem(
					"Test Inner 1-1",
					widget.NewVBox(widget.NewLabel("Test Inner 1-1")),
				),
				widget.NewTabItem(
					"Test Inner 1-2",
					widget.NewVBox(widget.NewLabel("Test Inner 1-2")),
				),
			),
		)),
		widget.NewTabItem("Tab 2", widget.NewVBox(
			widget.NewRadio([]string{
				"D",
				"E",
			}, nil),
			widget.NewTabContainer(
				widget.NewTabItem(
					"Test Inner 2-1",
					widget.NewVBox(widget.NewLabel("Test Inner 2-1")),
				),
				widget.NewTabItem(
					"Test Inner 2-2",
					widget.NewVBox(widget.NewLabel("Test Inner 2-2")),
				),
			),
		)),
	)

	win.SetContent(tabs)
	win.ShowAndRun()
}
```

### Device (please complete the following information):
 - OS: Windows
 - Version: 10
 - Go version: 1.14.1
 - Fyne version: ed30d6786124c2c34e9d454076901df3e3830ca8

**Comment by andydotxyz** [comment]
Possibly related to #938?

**Comment by andydotxyz** [comment]
These issues are unrelated. #938 is fixed by the improved render code threading but this is not.

**Comment by andydotxyz** [comment]
This is now on develop for testing, thanks @toaster.
It will be released in 1.3
